corehq/apps/app_manager/translations.py

```diff
@@ -558,8 +558,13 @@ def get_markdown_node(text_node_):
         return text_node_.find("./{f}value[@form='markdown']")
 
     def get_value_node(text_node_):
-        return next(n for n in text_node_.findall("./{f}value")
-                    if 'form' not in n.attrib or n.get('form') == 'default')
+        try:
+            return next(
+                n for n in text_node_.findall("./{f}value")
+                if 'form' not in n.attrib or n.get('form') == 'default'
+            )
+        except StopIteration:
+            return WrappedNode(None)
```

Comment:
No that should be fine. I checked all calls of the function and they all expect a `WrappedNode` to be returned which may be empty.